routes/get-current-project.js

```diff
@@ -20,7 +20,7 @@ module.exports = function(config) {
 
     if(!user) {
       if(!req.session.project.remixId) {
-        sendResponse(res, project, DefaultProject.map(function(f) {
+        sendResponse(res, project, defaultProject.getAsBuffers("stay-calm").map(function(f) {
```

Comment:
Move "stay-calm" into an `env`, since you use it in multiple places.  Maybe `DEFAULT_PROJECT_TITLE`?